photon-ml/src/main/scala/com/linkedin/photon/ml/model/GAMEModel.scala

```diff
@@ -22,29 +22,37 @@ import org.apache.spark.storage.StorageLevel
 import scala.collection.Map
 
 /**
- * Representation of the generalized additive mixed effect (GAME) model
+ * Generalized additive mixed effect (GAME) model
  *
- * @param gameModels A (modelName -> model) map representation of the models that consist of the GAME model
+ * @param gameModels A (modelName -> model) map containing the models that make up the complete GAME model
  */
 class GAMEModel(gameModels: Map[String, DatumScoringModel]) extends DatumScoringModel {
 
+  require(gameModels != null)
```

Comment:
OK - I like to spell out pre-conditions, and I tend to think they can be needed later and can help in development, but it's case by case, and I agree that this one might be overkill. I'll retract.